### PR TITLE
docs(router): Update the basic router guide to use standalone

### PR DESCRIPTION
**Pull request by atscott**
This commit updates the basic router-tutorial guide to use standalone features.

### CODE REVIEW

File: aio/content/guide/router-tutorial.md [hunk 1]
```diff
@@ -117,17 +106,45 @@ Each route typically has two properties.
...
+  {path: 'crisis-list', component: CrisisListComponent},
+  {path: 'heroes-list', component: HeroesListComponent},
+```
+
+This routes list is an array of JavaScript objects, with each object defining the properties of a route.
+
+## Import `provideRouter` from `@angular/router`
+
+Routing lets you display specific views of your application depending on the URL path.
+To add this functionality to your sample application, you need to update the `app.config.ts` file to use the module, `provideRouter`.
```
Comment (eneajaho) [inline aio/content/guide/router-tutorial.md:129]:
```suggestion
To add this functionality to your sample application, you need to update the `app.config.ts` file to use the router providers, `provideRouter`.
```

File: aio/content/guide/router-tutorial.md [hunk 2]
```diff
@@ -117,17 +106,45 @@ Each route typically has two properties.
...
+You import this module from `@angular/router`.
```
Comment (eneajaho) [inline aio/content/guide/router-tutorial.md:130]:
```suggestion
You import this provider function from `@angular/router`.
```

**Review by MarkTechson — APPROVED** [review]
LGTM 🚀

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 3dc74f3ac5ca0502ad13641effcede08dcfabbf7.

**Pull request closed without merging**